features/admin.branding.v1/components/preview/sign-in-box/fragments/push-auth-fragment.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import React, { FunctionComponent, ReactElement } from "react";
+import { CustomTextPreferenceConstants } from "../../../../constants/custom-text-preference-constants";
+import useBrandingPreference from "../../../../hooks/use-branding-preference";
+
+/**
+ * Proptypes for the push authentication fragment of login screen skeleton.
+ */
+export type PushAuthFragmentInterface = IdentifiableComponentInterface;
+
+/**
+ * PushAuthFragment fragment component for the branding preview of Sign In box.
+ *
+ * @param props - Props injected to the component.
+ * @returns PushAuthFragment fragment component.
+ */
+const PushAuthFragment: FunctionComponent<PushAuthFragmentInterface> = (
+    props: PushAuthFragmentInterface
+): ReactElement => {
+    const { ["data-componentid"]: componentId = "branding-preview-push-auth-fragment" } = props;
```

Comment:
Lets move the prop destructuring directly to the function parameters.

Reply:
Fixed with a0c8ecf